docs/models/data-types.mdx

```diff
@@ -222,18 +226,36 @@ For UUIDs, use `DataTypes.UUID`. It becomes the `UUID` data type for PostgreSQL
 
 ### Built-in Default Values for UUID
 
-Sequelize can generate UUIDs automatically for these attributes, simply use `DataTypes.UUIDV1` or `DataTypes.UUIDV4` as the default value:
+Sequelize can generate UUIDs automatically for these attributes, simply use [`sql.uuidV1` or `sql.uuidV4`](../querying/raw-queries.mdx#sqluuidv4--sqluuidv1) as the default value:
 
 ```javascript
 MyModel.init({
   myUuid: {
-    type: DataTypes.UUID,
-    defaultValue: DataTypes.UUIDV4, // Or DataTypes.UUIDV1
+    type: DataTypes.UUID.V4,
+    defaultValue: sql.uuidV4, // Or sql.uuidV1
   },
 });
 ```
 
-<JsDefaultCaution />
+In supported dialects, Sequelize will set the default value to the appropriate function, as shown in the table below.
+These dialects can also use these two functions in migrations.
+
+In all other dialects, Sequelize will generate the UUID value itself, in JavaScript.
+This means that they cannot use these two functions in migrations.
+
+<UuidSupportTable />

```

Comment:
Do we want to add footnotes for postgres <13 and mysql <8.0.31?

Reply:
Added info for postgres < 13 here, and added a note for mysql < 8.0.13 in the defining-models page